### PR TITLE
Feature 16 hungry people in order backend

### PR DESCRIPTION
**Pull request by Wojtasik**
Backend for adding and removing hungry people in order.
To add one, send post request to `/api/order/addMeal`. Example:

``` json
{
    "orderId": "56ed69f5fe8bb98144d5bcf3",
    "hungryGuy": "Pat",
    "name": "burger",
    "cost": "20"
}
```

To remove one, send delete request to: `/api/order/removeMeal`. Example:

``` json
{
    "orderId": "56ed69f5fe8bb98144d5bcf2",
    "id": "56ed69f5fe8bb98144d5bcf5"
}
```

### CODE REVIEW

File: src/lib/errorsHandler.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+'use strict';
```
Comment (kjendrzyca) [inline src/lib/errorsHandler.js]: I don't see the point in creating "lib" folder just for one file
EDIT:
nvm, I see what you did there, but still what does "lib" even mean? we need to come up with better name

File: src/router.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ const orders = require('./controllers/orders');
 
 module.exports = (app) => {
     app.get('/api/orders', orders.list);
+    app.post('/api/order', orders.create);
     app.get('/api/orders/:id', orders.get);
-    app.post('/api/orders', orders.create);
+    app.post('/api/order/addMeal', orders.addMeal);
+    app.delete('/api/order/removeMeal', orders.removeMeal);
```
Comment (kjendrzyca) [inline src/router.js]: what do you think about more restful api, like api/order/meal (POST and DELETE)

**Comment by kjendrzyca** [comment]
looks ok, some minor comments, let's agree on something and I will merge it

File: src/router.js (hunk 2 above)
Comment (Wojtasik) [inline src/router.js]: Sure! Good one.

File: src/lib/errorsHandler.js (hunk 1 above)
Comment (Wojtasik) [inline src/lib/errorsHandler.js]: i've read somewhere about lib/libs directory for this kind of files, probably it was on stack.

**Comment by Wojtasik** [comment]
Wait with the merge until we figure the name for the directory. I'll make changes tomorrow.

File: src/lib/errorsHandler.js (hunk 1 above)
Comment (tarnas14) [inline src/lib/errorsHandler.js]: I've read you should use jquery for this type of problem, probably somewhere on stack

Comment (kjendrzyca) [inline src/lib/errorsHandler.js]: yeah http://www.doxdesk.com/img/updates/20091116-so-large.gif

Comment (Wojtasik) [inline src/lib/errorsHandler.js]: Cool! Any suggestions?

**Comment by Wojtasik** [comment]
Done.

**Pull request merged**